infrastructure/staking-mechanics/offenses-and-slashes.md

```diff
@@ -1,4 +1,179 @@
 ---
 title: Offenses and Slashes
-description: TODO
----
\ No newline at end of file
+description: Learn about how Polkadot discourages validator misconduct via an offenses and slashing system, including details on offenses and their consequences.
+---
+
+# Offenses and Slashes
+
+## Introduction
+
+In Polkadot's Nominated Proof-of-Stake (NPoS) system, validator misconduct is deterred through a combination of slashing, deactivation, and reputation penalties. Validators and nominators who stake tokens face consequences for validator misbehavior, which range from token slashes to restrictions on network participation.
+
+This page outlines the types of offenses recognized by Polkadot, including block equivocations and invalid votes, as well as the corresponding penalties. While some parachains may implement additional custom slashing mechanisms, this guide focuses on the offenses tied to staking within the Polkadot ecosystem.
+
+??? info "Deactivation"
```

Comment:
I'd say this seems important and we should expand it

Reply:
This also feels a bit weird to be here as a note without any mentioning of it beforehand